garageScript/c0d3-app
# Add exercise preview cards to DOJO exercises page

This pull request adds a new ExercisePreviewCard component which gets used on the DOJO exercises page.
The exercise preview cards are populated with mock data on the frontend, but this will be removed once the backend returns DOJO exercise preview data.

![screenshot](https://user-images.githubusercontent.com/7637655/189548557-2bbeba5b-84af-4096-9360-8e8cdfecf4b4.png)


This change doesn't affect any user-facing pages since there are no links to the /exercises/:lessonSlug page right now.

How to test:
* Go to /exercises/js0
* Make sure everything looks good

This pull request is a part of https://github.com/garageScript/c0d3-app/issues/2253.
